services/event-application-service/src/pubsub_controller/pubsub_subscribers.cc

```diff
@@ -0,0 +1,98 @@
+#include "pubsub_subscribers.h"
+
+#include "glog/logging.h"
+#include "pubsub_controller/pubsub_controller.h"
+#include "queries/domain.h"
+#include "commands/domain.h"
+
+namespace pubsub = ::google::cloud::pubsub;
+
+// namespace eas::pubsub_controller {
+
+// void CreateFormSub::operator()(google::cloud::pubsub::Message const &msg,
+//                                google::cloud::pubsub::AckHandler ack) {
+//   // add simple validation.
+//   LOG(INFO) << msg;
+//   auto cmd = commands::CreateForm{msg.data()};
+//   std::move(ack).ack();
+//   handler_->Handle(cmd);
+// }
+
+// void CreateResponseSub::operator()(google::cloud::pubsub::Message const &msg,
+//                                    google::cloud::pubsub::AckHandler ack) {
+//   // add simple validation.
```

Comment:
Graveyard?

Reply:
Yeap. Let's keepit it for now. It is not compiled, and I will remove it with next pr.